src/shared/common/SatImage.js

```diff
@@ -105,6 +105,7 @@ export default class SatImage extends React.Component<Props, State> {
                         results={0}
                         style={styles.imageBackground}
                         tile={task}
+                        source={source}
```

Comment:
Since this is a shared component, did you check that adding this prop has no weird side effects on the other project types?

Reply:
So, I mapped a normal project. But to be honest didn't have a proper way to test for things. What would you suggest?

For the other project types, e.g. in the normal one, the `source` attribute is not used. So unless there are side effects when you don't pass an argument, then I think we are fine.

Reply:
Not much else I think. if the usual projects work, the tutorials as well, then it should be fine. Oh and `eslint / flow` are happy as well, so all good I think.

Reply:
okay, yeah also checked the tutorial.